### PR TITLE
Update Symfony to 6.3 and PHP 8.2

### PR DESCRIPTION
**Pull request by jskowronski39**
This will:
- Update Symfony to 6.3
- Update PHP to 8.2
- Resolve all deprecations and other issues caused by the upgrade

### CODE REVIEW

File: config/packages/doctrine_migrations.yaml [hunk 1]
```diff
@@ -3,10 +3,4 @@ doctrine_migrations:
         # namespace is arbitrary but should be different from App\Migrations
         # as migrations classes should NOT be autoloaded
         'DoctrineMigrations': '%kernel.project_dir%/migrations'
-    enable_profiler: '%kernel.debug%'
-
-    # This is required to ensure BC with previous table name
-    # https://github.com/doctrine/migrations/blob/3.0.x/UPGRADE.md
-    storage:
-        table_storage:
-            table_name: migration_versions
```
Comment (veteran29) [inline config/packages/doctrine_migrations.yaml:12]: Won't this cause issues on existing deployments?

File: webpack.config.js [hunk 2]
```diff
@@ -54,12 +64,15 @@ Encore
     // uncomment if you use TypeScript
     //.enableTypeScriptLoader()
 
+    // uncomment if you use React
+    //.enableReactPreset()
+
     // uncomment to get integrity="..." attributes on your script & link tags
     // requires WebpackEncoreBundle 1.4 or higher
-    .enableIntegrityHashes(Encore.isProduction())
+    //.enableIntegrityHashes(Encore.isProduction())
```
Comment (veteran29) [inline webpack.config.js:72]:
```suggestion
    .enableIntegrityHashes(Encore.isProduction())
```

**Review by veteran29 — CHANGES_REQUESTED** [review]

File: config/packages/doctrine_migrations.yaml (hunk 1 above)
Comment (jskowronski39) [reply config/packages/doctrine_migrations.yaml]: It will, restored.

File: webpack.config.js (hunk 2 above)
Comment (jskowronski39) [reply webpack.config.js]: Enabled

**Review by veteran29 — APPROVED** [review]

**Pull request merged**